contributing/topics/guide-breaking-changes.md

```diff
@@ -340,7 +340,7 @@ This is a breaking change as Terraform should not trigger a plan between minor v
 
```

Comment:
Should line 339 state something along the lines of "Instead, what we can do is use the major release feature flag as shown in the example below" instead of "...add a TODO next to the `Default` tag..."?